app/main/actions/profile-actions.js

```diff
@@ -48,3 +50,21 @@ export function completionsFor(text: string,
     },
   };
 }
+
+export function createBrowserWindow(browserWindow: electron.BrowserWindow): ProfileAction {
+  return {
+    type: profileActionTypes.DID_CREATE_BROWSER_WINDOW,
+    payload: {
+      browserWindow,
+    },
+  };
+}
+
+export function closeBrowserWindow(browserWindow: electron.BrowserWindow): ProfileAction {
```

Comment:
@bgrins and I discussed this on Slack.  We're fairly sure the command separation will go away over time, since it's not carrying it's weight.

As for why closing is a command, but opening isn't -- that's just work in progress.  I expect all of these things to be commands soon enough.